### PR TITLE
[950] Courses summary spike [WIP]

### PR DESCRIPTION
**Pull request by benilovj**
### Context
A spike implementation of https://manage-courses-prototype.herokuapp.com/history/status-logic-before-transition

### CODE REVIEW

**Comment by fofr** [comment]
In https://manage-courses-prototype.herokuapp.com/quiz:
We also have "Opens on [date]" for applications.

**Comment by defong** [comment]
looks about right to me, from data point of view

**Comment by dankmitchell** [comment]
Looks good to me. It would be nice to clearly define the v1 and v2 serialisers but the logic is good and very clear 😍

File: app/controllers/api/v2/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Api
+  module V2
+    class CoursesController < ApplicationController
+      skip_before_action :authenticate
+
+      def index
```
Comment (dankmitchell) [inline app/controllers/api/v2/courses_controller.rb]: Should this be a `provider#show` instead?

**Comment by benilovj** [comment]
> Looks good to me. It would be nice to clearly define the v1 and v2 serialisers but the logic is good and very clear 😍

We could move the existing v1 serialisers into a `UCAS` namespace (since the representation is quite UCAS-specific). That way we could reclaim the default serializer names.

**Comment by dankmitchell** [comment]
> We could move the existing v1 serialisers into a `UCAS` namespace (since the representation is quite UCAS-specific). That way we could reclaim the default serializer names.

Great idea @benilovj. I'm onboard with that

**Comment by dankmitchell** [comment]
[Create api/v2 for manage courses frontend](https://trello.com/c/ChXP2RQm/950-create-api-v2-for-manage-courses-frontend)

**Comment by defong** [comment]
> In https://manage-courses-prototype.herokuapp.com/quiz:
> We also have "Opens on [date]" for applications.

Covered in this commit
https://github.com/DFE-Digital/manage-courses-backend/pull/93/commits/7b82d80ea6b501b7b3c5dd14502503310b94d972

Basically `Applications` is a ui concern of which backend needs to provide enough to determine what to exactly show ie `Open`|`Closed`|`Date`|`Vacancy`

**Comment by benilovj** [comment]
Closing, will reopen with better branch name and cleaner PR.

**Comment by benilovj** [comment]
Reopened as #118.

**Pull request closed without merging**